main.c

```diff
@@ -286,22 +289,22 @@ void setTextTexture(TEXT_TEXTURE *txtText, TTF_Font *font, char *text) {
 
 void updatePoints(int n) {
   gPoints += n;
-  sprintf(pointsText, "%d", gPoints);
+  debugPrint(pointsText, "%d", gPoints);
```

Comment:
This breaks the game. You *must* use `sprintf`.

- `sprintf` prints text to a variable (critical for game logic).
- `printf` prints text to the terminal (typically useless / for warning / error / ... output or something - on nxdk the terminal isn't hooked up).
- `fprintf` prints text to file (either actual file or terminal - on nxdk the terminal isn't hooked up).